tests/test_data_model.py

```diff
@@ -71,13 +71,30 @@ def test_lstm_simple(self):
         # Create from model
         LSTMStateTransitionModel(m.model, output_keys = ['x'])
 
+        # Test pickling model m
+        import pickle
+        pickled_m = pickle.dumps(m)
+        m2 = pickle.loads(pickled_m)
```

Comment:
Just to verify the above, I changed lines 76 and 77 to 
```
pickled_m = pickle.dump(m, open('model.pkl', 'wb')) # dumps --> dump
m2 = pickle.load(open('model.pkl', 'rb')) # loads --> load
```
and received the same error.